pantheon-bundle/src/main/java/com/redhat/pantheon/servlet/module/PublishDraftVersion.java

```diff
@@ -91,7 +91,7 @@ protected void doRun(SlingHttpServletRequest request, PostResponse response, Lis
             versionToRelease.get()
                     .metadata().getOrCreate()
                     .datePublished().set(Calendar.getInstance());
-            changes.add(Modification.onModified(module.getPath()));
+            changes.add(Modification.onModified(module.getPath())); 
```

Comment:
Ah, same thing here, if there are no meaningful changes in this file then please revert it.

Reply:
Did the changes here as well.
I don't see any changes apart from an extra space, which I have taken care of.